plugins/importexport/native/filter/SubmissionFileNativeXmlFilter.inc.php

```diff
@@ -121,12 +120,33 @@ function createSubmissionFileNode($doc, $submissionFile) {
 		}
 
 		$submissionFileNode->appendChild($revisionNode);
-
 		// Embed the file contents
-		$embedNode = $doc->createElementNS($deployment->getNamespace(), 'embed', base64_encode(file_get_contents($submissionFile->getFilePath())));
-		$embedNode->setAttribute('encoding', 'base64');
-		$revisionNode->appendChild($embedNode);
 
+		if (array_key_exists('no-embed', $this->opts)) {
+			$hrefNode = $doc->createElementNS($deployment->getNamespace(), 'href', );
```

Comment:
I think that trailing comma is a syntax error...

Reply:
fixed now: https://github.com/pkp/pkp-lib/pull/5524/files#diff-8c95f7fe60b5739a2bed1f1c4443a005R126